packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -684,34 +718,44 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
         alignElement.ariaLabel = this.ctx.localeManager.t(map[group]);
     }
 
+    private expandConfig(button: HTMLButtonElement, options: ButtonConfiguration): ToolbarButtonConfig {
```

Comment:
Function name should reference button, as there's a lot of config and util functions in this class.

Reply:
ACK.

Reply:
Done.